SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -775,17 +800,22 @@ void USpatialReceiver::HandleActorAuthority(const Worker_AuthorityChangeOp& Op)
 					Actor->OnAuthorityLost();
 				}
 			}
+
+			if (Authority != WORKER_AUTHORITY_AUTHORITY_LOSS_IMMINENT)
+			{
+				EventTracer->TraceEvent(FEventAuthorityLossImminent(Actor->Role, Actor), { Op.span_id });
```

Comment:
It looks like this is the only real change to this function and we definitely don't care about auth loss imminent.
If there isn't something significant here I'd say we should leave this as it was because this is adding noise to a large change log but is going to be removed by subsequent PRs.

Reply:
Cool. Yeah you are right that I probably should not have made these additional changes in this PR. I think it now might be more work to revert them. So perhaps we just keep them there? Up to you.